lib/backend/lite/lite.go

```diff
@@ -337,16 +332,43 @@ func (l *Backend) createSchema() error {
 		}
 	}
 
+	for table, column := range map[string]string{"kv": "revision", "events": "kv_revision"} {
+		migrationRequired, err := l.performRevisionMigration(table, column)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+		if migrationRequired {
+			query := fmt.Sprintf(`ALTER TABLE %s ADD COLUMN %s TEXT NOT NULL DEFAULT "";`, table, column)
```

Comment:
Do we have a general idea of how long this will take?

I also just realized that we should do this in a read-write transaction so other instances of the backend (if any) are not going to get confused as they first check that the column doesn't exist and then fail to add a column that was added by someone else.